legion/engine/rendering/pipeline/default/stages/particlebatchingstage.cpp

```diff
@@ -5,15 +5,24 @@ namespace legion::rendering
 {
     void ParticleBatchingStage::setup(app::window& context)
     {
-
+        create_meta<sparse_map<material_handle, sparse_map<model_handle, std::pair<std::vector<math::mat4>, std::vector<size_type>>>>>("particle batches");
     }
 
     void ParticleBatchingStage::render(app::window& context, camera& cam, const camera::camera_input& camInput, time::span deltaTime)
     {
-        static id_type batchesId = nameHash("mesh batches");
-        auto* batches = get_meta<sparse_map<material_handle, sparse_map<model_handle, std::pair<std::vector<id_type>, std::vector<math::mat4>>>>>(batchesId);
+        static id_type batchesId = nameHash("particle batches");
+        auto* batches = get_meta<sparse_map<material_handle, sparse_map<model_handle, std::pair<std::vector<math::mat4>, std::vector<size_type>>>>>(batchesId);
```

Comment:
yeah i feel like maybe the size difference betwen uint (32bit) and size_type (64bit) is causing a bug somewhere